config/environments/production.rb

```diff
@@ -36,7 +36,7 @@ module OptimizerWrapper
   HEURISTICS = %w[path_cheapest_arc global_cheapest_arc local_cheapest_insertion savings parallel_cheapest_insertion first_unbound christofides].freeze
   WEEKDAYS = %i[mon tue wed thu fri sat sun].freeze
   DEMO = Wrappers::Demo.new(tmp_dir: TMP_DIR, threads: 4)
-  VROOM = Wrappers::Vroom.new(tmp_dir: TMP_DIR, threads: 4)
+  VROOM = Wrappers::Vroom.new(tmp_dir: TMP_DIR, threads: 1)
```

Comment:
Do you want to leave it 4 for tests and dev (or is it already 0)?

I would make the same thing for other environments so that we don't have a different (even better) performance when it is not production.

Reply:
We did not specify number of threads in test and development but it would make sense to have same limit everywhere, yes.